packages/suite/src/actions/suite/__tests__/analyticsActions.test.ts

```diff
@@ -49,7 +49,7 @@ const oldWindowLocation = window.location;
 describe('Analytics Actions', () => {
     beforeAll(() => {
         // eslint-disable-next-line global-require
-        require('@suite-utils/random');
+        require('@trezor/utils');
```

Comment:
are you sure this is even needed? I tried to remove it and test still works

Reply:
Not needed, removed :+1: